src/RenderWebGL.js

```diff
@@ -1040,11 +1040,18 @@ class RenderWebGL extends EventEmitter {
         const scratchY = this._nativeSize[1] * ((y / this._gl.canvas.clientHeight) - 0.5);
 
         const gl = this._gl;
-        twgl.bindFramebufferInfo(gl, this._queryBufferInfo);
 
         const bounds = drawable.getFastBounds();
         bounds.snapToInt();
 
+        // Use a new bufferInfo since this._queryBufferInfo is limited to 480x360
+        const attachments = [
+            {format: gl.RGBA},
+            {format: gl.DEPTH_STENCIL}
+        ];
+        const bufferInfo = twgl.createFramebufferInfo(gl, attachments, bounds.width, bounds.height);
```

Comment:
This could fail if the bounds are too big, and even if it succeeds it could eat up a lot of memory. I would suggest checking the max texture size with something like `const maxTextureSize = gl.getParameter(gl.MAX_TEXTURE_SIZE);` and then using something like `Math.min(2048, bounds.width, maxTextureSize)` and similar for `bounds.height`. The 2048 is in there to provide a reasonable maximum even if you have a huge sprite and an ambitious GPU :)

Reply:
I would also add a `gl.checkFramebufferStatus` check on the new `bufferInfo` and, if that fails, fall back to using the query buffer as a last resort.